common/app/com/commercetools/sunrise/play/SunriseHttpFilters.java

```diff
@@ -9,13 +11,13 @@
  * Based on play.mvc.http.DefaultHttpFilters with fix https://github.com/playframework/playframework/pull/6238, which will be released with Play 2.5.5.
  * REMOVE AS SOON AS IT IS RELEASED!
  */
-public class BasicAuthHttpFilters implements HttpFilters {
+public class SunriseHttpFilters implements HttpFilters {
```

Comment:
@ccsalazarr-commercetools I've finally renamed it to `SunriseHttpFilters`, as I saw it is unique per project (or at least that's how it is documented). I've also move it outside from `basicauth`, since it's not related to it.